catherine_rossi/email_validation_assignment/server.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask, request, redirect, render_template, session, flash
+from mysqlconnection import MySQLConnector
+app = Flask(__name__)
+mysql = MySQLConnector(app,'friends_db')
+
+@app.route('/') #homepage
+def index():
+    return render_template('index.html')
+
+@app.route("/emails", methods=["POST"]) 
+def insert():
+    #validating emails using html5
```

Comment:
don't rely solely on HTML5 to validate _anything_ that goes in your DB.  It's okay to have validations on the front end as long as those validations are duplicated on your server.  The reason being that front-end validations can easily be bypassed.